Admin/Extension/MenuNodeReferrersExtension.php

```diff
@@ -21,20 +21,47 @@
  */
 class MenuNodeReferrersExtension extends AdminExtension
 {
+    /**
+     * @var string
+     */
+    private $formTab;
+
+    /**
+     * @var string
+     */
+    private $formGroup;
+
+    public function __construct($formGroup = 'form.group_menu', $formTab = 'form.tab_menu')
+    {
+        $this->formGroup = $formGroup;
+        $this->formTab = $formTab;
+    }
+
     public function configureFormFields(FormMapper $formMapper)
     {
+        if ($formMapper->hasOpenTab()) {
+            $formMapper->end();
+        }
```

Comment:
if that `with` thing works both for sonata 2.3 and 2.4, we could switch
to that, default to tabs but allow configuration so people can put it in
columns or tabs as they want. that could be neat.